Check SQL Logins for Weak Passwords - updates

SQLAdrian opened the pull request:
Added more core passwords from danielmiessler/SecLists/
Do basic permutation steps on core passwords
Do a connection count in the output screen.
Tested on SQL 2012 and 2019

SQLAdrian committed:
2022-03-12, Adrian Sullivan, addded more deault passwords from https://github.com/danielmiessler/SecLists/tree/master/Passwords
Added replace permuration on main password list for common characters replace 'a' with '@', '3' with '#', 's' with '$', 'o' with '*'

SQLAdrian committed:
Check weak accounts curently connecting and filter better on modifiers

SQLAdrian committed:
updated password estimat

EitanBlumin commented:
Thank you for your contribution, Adrian!

I'll get this sorted out to remove any duplicates and then merge.

EitanBlumin committed:
Added new parameters and optimized performance

also removed duplicates to rely on generated passwords rather than explicitly inserted passwords

EitanBlumin commented:
I made several optimizations and I'm merging this into the main branch.

SQLAdrian commented:
Legend. Thanks Eitan. I love your work.
